src/main/java/com/notification/controller/NotificationController.java

```diff
@@ -70,4 +70,11 @@ public ResponseEntity<ResponseModel<InvitationStatusResponse>> updateStatus(
         log.info("Endpoint: /put update status ends with update status: {}", response.getStatus());
         return ResponseEntity.ok(response);
     }
+
+    @PostMapping(value = "/send-registration-mail", produces = MediaType.APPLICATION_JSON_VALUE)
+    public void sendConfirmationMail(@RequestBody InviteRequest inviteRequest) {
```

Comment:
Remove inviteRequest and use PathVariable as sentOnEmail

Reply:
I am using two variables here, which are user email and user name, which is why I am using the existing InviteRequest.

Reply:
Ok